site/content/entry/yuzu-progress-report-mar-2020/index.md

```diff
@@ -0,0 +1,131 @@
++++
+date = "2020-04-19T04:00:00-03:00"
+title = "Progress Report March 2020"
+author = "GoldenX86"
+forum = 217502
++++ 
+
+How are you doing in this quarantine, yuz-ers? We hope you are all staying in your homes safe and sound.
+Today, we are bringing you a report on a few changes from the past month for both the Early Access and regular Mainline versions of yuzu.
+<!--more-->
+
+We have on our plate a couple of bug fixes, some graphical improvements, the first “performance win” for Intel GPUs, a really serious case of serendipity, and some groundwork for amazing improvements that will be coming to you in a “near-ish” future.
+
+## Bug fixes and improvements
+
+After some intensive debugging by [Rodrigo](https://github.com/ReinUsesLisp), an OpenGL rendering bug in ARMS (that was with us since the game started to boot), was solved by simply updating our buildbot VM from Visual Studio 2017 to the latest 2019 version.
+Thanks to [DarkLordZach](https://github.com/DarkLordZach) for the change!
+
+PSA - Please update to or install the latest [Visual C++ 2019 x64 redistributable](https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads), or else you won’t be able to open yuzu, as it will complain that the library VCRUNTIME140.dll cannot be found.
+
+{{< single-title-imgs
+    "Better, right?"
+    "./03.png"
+    "./04.png"
+  >}}
+
+perillamint and [bunnei](https://github.com/bunnei) stubbed (this means to send a fake `everything’s OK` signal so the game can continue) some VR related services.
+These changes are needed for version 1.6.0 of The Legend of Zelda: Breath of the Wild, 1.3.0 of Super Mario Odyssey, and versions 3.1.0 and above of Super Smash Bros. Ultimate.
+
+Commercial games aren't everything.
+[Rodrigo](https://github.com/ReinUsesLisp) decided to give homebrew lovers and old-school gamers something to play with, by implementing some attributes from the old OpenGL 1.X and 2.X days.
+It’s now possible to run `Xash3D`, so you can frag in Counter-Strike, or follow G-Man’s orders in Half-Life!
+Currently, this needs to be manually built with newer dependencies, but that should be solved for good in the near future.
+
+Margen67 and [Morph](https://github.com/Morph1984) wanted to improve the graphics for games and so they added Anisotropic Filtering (AF) to yuzu.
+With this feature in place, distant textures now look much sharper, improving game quality even over what the original Switch does! 
+Games like Pokémon Sword, Pokémon Shield, and Fire Emblem Three Houses benefit a lot from this.
```

Comment:
I believe the removing the period after this sentence and replacing it with a semicolon will make the paragraph more cohesive.

change to `from this; however, some other`